src/services/registration.js

```diff
@@ -0,0 +1,36 @@
+const regModel = require('../models/registrationExample');
+const checkName = require('../services/checkUserName');
+const checkPsw = require('../services/checkPassword');
+const dbCheck = require('../services/regCheckDB');
+const users = require('../models/userDB');
+
+const createUser = () => new Promise((resolve) => {
+  const temp = {
+    userId: '',
+    username: '',
+    password: '',
+    heroes: [],
+  };
+  if (checkName.CheckUserName(regModel.User.username) && checkPsw.CheckPassword(regModel.User.password)) {
```

Comment:
When pinging the endpoint with Postman, and sending nothing in the request body, this passes the request on and it fails at the next IF statement, responding to Postman with a 'Passwords don't match' message